vnext/ReactUWP/Utils/ValueUtils.cpp

```diff
@@ -180,6 +181,31 @@ DateTimeToDynamic(winrt::DateTime dateTime, int64_t timeZoneOffsetInSeconds) {
   return strDateTime;
 }
 
+// In UWP apps, the string provided to FontFamily must be either a system-installed font name or a
+// a special path string to a font file bundled with the app (e.g. "/Assets/MyFont.ttf#MyFont").
+// Since this convention for custom fonts is not portable, the following code provides a mechanism for
+// app developers to register their custom font paths with aliases and use those aliases in React for any
+// FontFamily styles.
+
+static FontManager s_FontManager{};
+
+REACTWINDOWS_API_(void)
+SetFontPath(
+    const std::wstring &fontFamily,
```

Comment:
>fontFamily [](start = 24, length = 10)

rename to fontFamilyName? here and below since these are not FontFamily instances but names